OutOfSchool/OutOfSchool.DataAccess/OutOfSchool.DataAccess.csproj

```diff
@@ -23,15 +23,15 @@
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="Microsoft.AspNetCore.DataProtection.EntityFrameworkCore" Version="5.0.2" />
-      <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="5.0.2" />
-      <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.2" />
-      <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="5.0.2">
+      <PackageReference Include="Microsoft.AspNetCore.DataProtection.EntityFrameworkCore" Version="5.0.11" />
+      <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="5.0.11" />
+      <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.11" />
+      <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="5.0.11">
         <PrivateAssets>all</PrivateAssets>
         <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       </PackageReference>
-      <PackageReference Include="Microsoft.EntityFrameworkCore.Proxies" Version="5.0.2" />
-      <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.2" />
+      <PackageReference Include="Microsoft.EntityFrameworkCore.Proxies" Version="5.0.11" />
+      <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.11" />
```

Comment:
do we need SQL server dependency? MySQL maybe?

Reply:
p.s. can you add an example connection string to app settings for development?

Reply:
fixed